app/Controllers/UpdateAddressController.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\App\Controllers;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use WMDE\Fundraising\Frontend\Factories\FunFunFactory;
+
+/**
+ * @license GNU GPL v2+
+ */
+class UpdateAddressController {
```

Comment:
Please rename it to `UpdateDonorController`, as updating addresses will become its own thing which might or might not overlap with this.